### DISCUSSION TITLE
Mobile APP?

### DISCUSSION THREAD

**Discussion by lehhair** [misskey-dev/misskey#16844] [Q&A]
## Summary

Is there a mobile APP like mastodon that can use the self-built misskey?

**Comment by cobalto** [comment]
What do you mean by self-built?

For third-party clients, there are some
- [takesama](https://takesama.com/) (Android/iOS)
- [Miria](https://github.com/shiosyakeyakini-info/miria) (Android/iOS)
- [Kimis](https://apps.apple.com/us/app/kimis-a-client-for-misskey/id1667275125?platform=mac) (iOS/macOS)
- [Milktea](https://play.google.com/store/apps/details?id=jp.panta.misskeyandroidclient&hl=en_US) (Android)

**Comment by lehhair** [comment]
> What do you mean by self-built?
> 
> For third-party clients, there are some
> - [takesama](https://takesama.com/) (Android/iOS)
> - [Kimis](https://apps.apple.com/us/app/kimis-a-client-for-misskey/id1667275125?platform=mac) (iOS/macOS)
> - [Miria](https://play.google.com/store/apps/details?id=info.shiosyakeyakini.miria) (Android)
> - [Milktea](https://play.google.com/store/apps/details?id=jp.panta.misskeyandroidclient&hl=en_US) (Android)

When I say self-built, I mean my own misskey server. I just want a mobile APP. Thank you very much for your reply.

**Comment by cobalto** [comment]
All of them should work, set the server is usually the first step for add an account

**Comment by tai-cha** [comment]
Miria also supports iOS

**Comment by cobalto** [comment]
> Miria also supports iOS

You're right.
Fixed the link and description :)
